### PR TITLE
Welcome Page / Users Table

### PR DESCRIPTION
**Pull request by megroth2**
- added route, controller, and view for welcome page
- added routes for sign up/login buttons (had to add these so the welcome page would display properly on my local server)
- added tests
- added Users table and model
- added functionality to log out 
- installed bcrypt
- uploaded chicken tracker logo

I'm curious if we could implement react or something to make the UI prettier. I'll just build them as craigslist pages for now, but maybe this is something we can dig into over the weekend or whenever we have extra time!

### CODE REVIEW

File: .gitignore
Comment (s2an) [inline .gitignore:1]: Good catch!

File: db/migrate/20240222034449_create_users.rb
Comment (s2an) [inline db/migrate/20240222034449_create_users.rb:1]: We might have discussed this already, but would shelters be a string? Or would it be associated by shelters and need it's Id?
As in, once the shelter is created, then it has an id which is linked to the user?

**Comment by s2an** [comment]
Great work! It sure is weird to see what else is needed that is required for your user story. I was running in to that with creating my animal form. Trying to do TDD and working backwards on the associations required so much work on creating shelters (as a poro) and also the facade and service for the connection...it became confusing.

TL;DR - Writing user stories is an art in itself!
